### PR TITLE
Td 114 cucumber

### PR DESCRIPTION
**Pull request by ericescalante**
**JIRA Ticket**
https://factorymedia.atlassian.net/browse/TD-114

**Summary**
Proposed Cucumber entry for our standards doc.

### CODE REVIEW

File: testing/cucumber/README.md [hunk 1]
```diff
@@ -0,0 +1,279 @@
...
+
+Install the n package:
+`npm install -g n`
+
+Get the latest stable version of node:
+`n install latest`
+
+Make sure you're using the latest version of npm:
+`npm upgrade -g npm`
+
```
Comment (minsikzzang) [inline testing/cucumber/README.md:26]: I think you should mention how to run tests?

File: testing/cucumber/README.md [hunk 2]
```diff
@@ -0,0 +1,279 @@
...
+|     step2-definitions.coffee
+|   +-- support
+|     world.coffee
+|   feature-name1.feature
+|   feature-name2.feature
+|   feature-name3.feature
+```
+
+### Feature files
+These files are written [Gherkin](https://github.com/cucumber/cucumber/wiki/Gherkin) and drive the automated tests by specifying scenarios under which we test the feature in question.
```
Comment (willhall88) [inline testing/cucumber/README.md]: "These files are written _IN_ [Gherkin]" 
;) just me being pedantic :)

**Comment by willhall88** [comment]
Agree with @minsikzzang comment. otherwise looks good! :+1: :smiley:

**Comment by minsikzzang** [comment]
omg 3 `Tyding`s :)

**Comment by minsikzzang** [comment]
Can you squash your `Tidying`s  to one nice `Tydying`? :)

**Comment by minsikzzang** [comment]
:+1:  if you squash them :)

**Pull request merged**